### PR TITLE
Upgrade junit5

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: consumer_with_junit5/mvnw [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (marcingrzejszczak) [inline consumer_with_junit5/mvnw]: https

File: consumer_with_junit5/mvnw.cmd [hunk 2]
```diff
@@ -7,7 +7,7 @@
 @REM "License"); you may not use this file except in compliance
 @REM with the License.  You may obtain a copy of the License at
 @REM
-@REM    https://www.apache.org/licenses/LICENSE-2.0
+@REM    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (marcingrzejszczak) [inline consumer_with_junit5/mvnw.cmd]: https

File: consumer_with_junit5/src/test/java/com/example/intoxication/IntoxicationControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+//remove::end[]
+@DirtiesContext
+class IntoxicationControllerTest {
+
+	@Autowired
+	MockMvc mockMvc;
+	@Autowired
+	IntoxicationController intoxicationController;
+	//remove::start[]
+	@StubRunnerPort("beer-api-producer")
```
Comment (marcingrzejszczak) [inline consumer_with_junit5/src/test/java/com/example/intoxication/IntoxicationControllerTest.java]: shouldn't it be a junit5 version or it doesn't matter?

File: consumer_with_junit5/mvnw (hunk 1 above)
Comment (OlgaMaciaszek) [reply consumer_with_junit5/mvnw]: that's `mvnw` file

File: consumer_with_junit5/mvnw.cmd (hunk 2 above)
Comment (OlgaMaciaszek) [reply consumer_with_junit5/mvnw.cmd]: that's `mvnw` file

**Pull request merged**